data/sources/00_areatree

```diff
@@ -28,12 +28,12 @@
       0108:E-Technik Verfügungsgebäude (N8):0108,n8
       0109:Reflexionsarmer Raum (N9):0109,n9
     02:Südgelände:suedgelaende
-      0201:StudiTUM Innenstadt (S1):0201,studitum-innenstadt
+      0201:StudiTUM Innenstadt (S1)|StudiTUM:0201,studitum-innenstadt
```

Comment:
As a clarification on how the last part is hadled (I dont want to look through the parser):
what does `studitum-innenstadt` in this case mean? https://nav.tum.sexy/building/studitum-innenstadt is not availiable..